android/app/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this file about? This is kind of a green background. Do we need this?
I think as icons need to be opaque we have to decide for a possible background color: https://stackoverflow.com/questions/47722958/transparent-adaptive-icon-background

Reply:
Hm.. I used the Android studio to generate the icons... 

I expected that all created files have a sense, but I'm not sad if we remove this if it is not necessary

Reply:
I think this is required but should be black or white. Depending on what looks better.

Reply:
If you create a new standard project in Android studio it's taking a default color. Maybe the generator did not work properly here. Every ic_launcher_background color can just set to #FBDA16 - we did this also with other #33FFFF-appearances.

Reply:
Ok, can someone help to change this?

I guess I will not be able to do this in the next week's :/

Reply:
As this is in the Release Epic it is not important to finish early.